recurring_contract/tests/test_recurring_contract.py

```diff
@@ -9,15 +9,16 @@
 #
 ##############################################################################
 
-from openerp.tests import common
 from datetime import datetime, timedelta
 from openerp import netsvc
+from openerp.addons.recurring_contract.tests.test_base_contract \
```

Comment:
Since you are in the same package, `from test_base_contract import test_base_contract` should be sufficient.